api/external/infra_proxy/request/orgs.proto

```diff
@@ -4,37 +4,53 @@ package chef.automate.api.infra_proxy.request;
 option go_package = "github.com/chef/automate/api/external/infra_proxy/request";
 
 message CreateOrg {
+    // Name of the Org.
     string name = 1;
+    // Admin user of the Org.
     string admin_user = 2;
+    // Admin Key of the Org.
     string admin_key = 3;
+    // ID of the Server.
     string server_id = 4;
+    // Projects of the Org.
     repeated string projects = 5;
 }
 
 message UpdateOrg {
+    // ID of the Org.
     string id = 1;
+    // Name of the Org.
     string name = 2;
+    // Admin user of the Org.
     string admin_user = 3;
-    string admin_key = 4;
-    string server_id = 5;
-    repeated string projects = 6;
+    // ID of the server.
+    string server_id = 4;
+    // Projects under which org is being associated.
```

Comment:
Would like to be consistent across APIs.
Also, this change should apply everywhere you have a `Projects` field (CreateOrg above as well as the `message Org` in orgs.proto, etc.)
```suggestion
    // List of projects this organization belongs to. May be empty.
```

Reply:
Thanks, it seem more readable...